ping/ping.c

```diff
@@ -337,6 +337,7 @@ main(int argc, char **argv)
 		.source6.sin6_family = AF_INET6,
 		.ni.query = -1,
 		.ni.subject_type = -1,
+    .rtt_precision = 0
```

Comment:
This has wrong indent, please fix it (and other wrong indents).

Reply:
Change spaces to tab characters for indent?

Reply:
I will do it.

Reply:
Done